lib/internal/bootstrap/switches/is_main_thread.js

```diff
@@ -6,8 +6,8 @@ const rawMethods = internalBinding('process_methods');
 // TODO(joyeecheung): deprecate and remove these underscore methods
 process._debugProcess = rawMethods._debugProcess;
 process._debugEnd = rawMethods._debugEnd;
-process._startProfilerIdleNotifier = rawMethods._startProfilerIdleNotifier;
-process._stopProfilerIdleNotifier = rawMethods._stopProfilerIdleNotifier;
+process._startProfilerIdleNotifier = () => {};
+process._stopProfilerIdleNotifier = () => {};
```

Comment:
@addaleax @BridgeAR I turned them into no-ops like #33138 does. I feel less strongly than Anna about emitting warnings but I also don't really see the point because the functionality has been broken-ish for a long time. Doing nothing is more correct than doing the wrong thing, IMO. :-)